src/main/java/com/uid2/shared/middleware/AttestationMiddleware.java

```diff
@@ -105,9 +105,15 @@ public void handle(RoutingContext rc) {
                     } else {
                         if (this.enforceJwt) {
                             LOGGER.info("JWT is required, but was not received. Attestation validation failed. SiteId: {}, Name: {}, Contact: {}", operatorKey.getSiteId(), operatorKey.getName(), operatorKey.getContact());
+                        } else {
+                            LOGGER.info("JWT is blank but is not required.");
```

Comment:
Maybe log the Operator info on this line and the attestation not successful line